### ISSUE TITLE
Sethekk Halls not resetting 

### ISSUE THREAD

**Issue by Masakakihara** [dalaranwow/dalaran-wow#5981]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: For 3 days now ive been locked in the same heroic Sethekk halls, ive seen this reported here years ago, but cant seem to find a workaround. Ive made ticket ingame. Yet no response

**Expected behaviour**: The heroic instance should reset daily

**Steps to reproduce the problem**:

1. Level a druid to 80
2. get the quest for summoning Anzu
3. Summon Anzu
4. Get locked for eternity

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Masakakihara** [comment]
So yesterday the dungeon reset on its own,
me hopefull the issue was fixed. But today i'm locked out again.
I don't know what's going on
